lib/live_debugger_refactor/services/callback_tracer/events.ex

```diff
@@ -0,0 +1,23 @@
+defmodule LiveDebuggerRefactor.Services.CallbackTracer.Events do
+  @moduledoc """
+  Temporary events for LiveDebuggerRefactor.Services.CallbackTracer.
+  """
+
+  use LiveDebuggerRefactor.Event
+
+  alias LiveDebugger.CommonTypes
+
+  defevent(TraceCalled,
+    id: neg_integer(),
```

Comment:
I'd be more specific here because it looks like id of event, not id of trace. Also I think it may be `0` actually, so it should be `non_positive_integer()` 🤔 
```suggestion
    trace_id: neg_integer(),
```